### DISCUSSION TITLE
Testnet

### DISCUSSION THREAD

**Discussion by Bboypaco** [community/community#34316] [Other Feature Feedback, Questions, & Ideas]
hello possible to have more bnb test ?

**Comment by benjiallen** [comment]
Can you help me understand what you mean by "bnb test"?

**Reply by Bboypaco** [reply]
bnb test to test applications before the public launch the bnb test and fictitious therefore no values

**Reply by Bboypaco** [reply]
the problem we only have the right to 0.5 bnb per day I would have liked to know if it is possible to have a lot at once

**Reply by benjiallen** [reply]
What's bnb? I'm not familiar with that acronym. Is that a GitHub product?

**Comment by Bboypaco** [comment]
Look

Le mer. 28 sept. 2022 à 14:25, Ben Allen ***@***.***> a
écrit :
